src/ui/dialog-vkbasalt.ui

```diff
@@ -0,0 +1,820 @@
+        <child>
+          <object class="AdwHeaderBar">
+            <property name="show-end-title-buttons">False</property>
+            <child type="start">
+              <object class="GtkBox">
+                <property name="spacing">6</property>
+                <child>
+                  <object class="GtkButton" id="btn_cancel">
+                    <property name="label" translatable="yes">_Cancel</property>
+                    <property name="use_underline">True</property>
+                    <property name="action_name">window.close</property>
+                  </object>
+                </child>
+              </object>
+            </child>
+            <child type="end">
+              <object class="GtkButton" id="btn_save">
+                <property name="label" translatable="yes">Save</property>
+                <style>
+                  <class name="suggested-action"/>
+                </style>
+              </object>
+            </child>
+          </object>
+        </child>
+        <child>
+          <object class="AdwPreferencesPage">
+            <child>
+              <object class="AdwPreferencesGroup">
+                <property name="title" translatable="yes">Default</property>
+                <child>
+                  <object class="AdwActionRow">
+                    <property name="activatable-widget">default</property>
+                    <property name="title" translatable="yes">Default Settings</property>
+                    <child>
+                      <object class="GtkSwitch" id="default">
+                        <property name="valign">center</property>
+                      </object>
+                    </child>
+                  </object>
+                </child>
+              </object>
+            </child>
+            <child>
+              <object class="AdwPreferencesGroup">
+                <property name="description" translatable="yes">Effects are applied according to the list order.</property>
+                <property name="title" translatable="yes">Effects</property>
+                <child>
+                  <object class="AdwExpanderRow" id="cas">
+                    <property name="show_enable_switch">True</property>
+                    <property name="title" translatable="yes">Contrast Adaptive Sharpening</property>
+                    <child>
+                      <object class="AdwPreferencesRow">
+                        <child>
+                          <object class="AdwActionRow">
+                            <property name="title" translatable="yes">Sharpness</property>
+                            <child>
+                              <object class="GtkMenuButton">
+                                <property name='valign'>center</property>
+                                <property name="tooltip-text" translatable="yes">Select</property>
+                                <property name="icon-name">info-symbolic</property>
```

Comment:
This icon is broken.
![image](https://user-images.githubusercontent.com/27908024/179656317-18da8c79-257f-4720-97d3-e7178102447d.png)

Reply:
Does it work now?